main/src/library/Int32.flix

```diff
@@ -0,0 +1,88 @@
+namespace Int32 {
+
```

Comment:
The code generator already makes this assumption. It makes things a lot easier since we're targeting the JVM.